Shared/Flows/Onboarding/CodeViewController.swift

```diff
@@ -12,7 +12,6 @@ import Parse
 import TMROLocalization
 import Combine
 
-#warning("Move main actor to the base view controller.")
```

Comment:
I'm going to wait for the final version of swift 5.5 before I do any more work on this. Not worth doing at the moment.